src/discordcr/mappings/rest.cr

```diff
@@ -15,5 +15,86 @@ module Discord
         pruned: UInt32
       )
     end
+
+    # A payload for the Modify Channel REST API call.
+    #
+    # Used instead of a tuple to control null emitting.
+    struct ModifyChannelPayload
+      JSON.mapping(
+        name: String?,
+        position: UInt32?,
+        topic: String?,
+        bitrate: UInt32?,
+        user_limit: UInt32?
+      )
+
+      def initialize(@name, @position, @topic, @bitrate, @user_limit)
```

Comment:
Looks like you added `nsfw` to the REST call but this `ModifyChannelPayload` object doesn't actually support it.